cfgov/login/urls.py

```diff
@@ -33,6 +33,12 @@
 else:
     urlpatterns = [
         re_path(r"^login/$", LoginView.as_view(), name="cfpb_login"),
+        # Override Wagtail's login URL.
+        # By overriding the path and name here, we avoid heavily customizing
+        # the wagtailadmin/login.html template.
+        re_path(
+            r"^admin/login/$", LoginView.as_view(), name="wagtailadmin_login"
+        ),
         # When SSO is not enabled, the Django login should redirect to our
         # /login URL, added above.
```

Comment:
See comment above about deprecating "/login/" - the GH PR interface won't let me suggest a diff for lines 44-46 here but we could do something like:

```py
re_path(
    r"^django-admin/login/$",
    RedirectView.as_view(pattern_name="wagtailadmin_login" query_string=True),
),
```

Reply:
Done in [0e09ebe](https://github.com/cfpb/consumerfinance.gov/pull/7637/commits/0e09ebe8fc529401f8f00a3751b6c0314cb8b03d).